src/configs/heroku.rs

```diff
@@ -0,0 +1,27 @@
+use crate::config::ModuleConfig;
+use serde::Serialize;
+use starship_module_config_derive::ModuleConfig;
+use std::collections::HashMap;
+
+#[derive(Clone, ModuleConfig, Serialize)]
+pub struct HerokuConfig<'a> {
+    pub format: &'a str,
+    pub symbol: &'a str,
+    pub style: &'a str,
+    pub disabled: bool,
+    pub app_aliases: HashMap<String, &'a str>,
+    pub account_aliases: HashMap<String, &'a str>,
+}
+
+impl<'a> Default for HerokuConfig<'a> {
+    fn default() -> Self {
+        HerokuConfig {
+            format: "[( $symbol on $app_name \\(via  $account\\) )]($style)",
+            symbol: " ",
```

Comment:
this is using nerdfont symbols. 

What is the policy here? Just use emojii in the default config, and provide an alternative nerdfont symbol in the nerdfont preset?